app/views/scaptimony_policies/_list.html.erb

```diff
@@ -8,23 +8,23 @@
   <% for policy in @policies %>
     <tr>
       <td>
-          <%= link_to policy.name, scaptimony_policy_dashboard_scaptimony_policy_path(:id => policy.id) %>
+        <%= display_link_if_authorized(policy.name.to_s, hash_for_scaptimony_policy_dashboard_scaptimony_policy_path(:id => policy.id)) %>
       </td>
       <td>
-          <% if !policy.scap_content.nil? %>
-            <%= link_to_if_authorized policy.scap_content.title,
-                  hash_for_edit_scaptimony_scap_content_path(:id => policy.scap_content) %>
-          <% end %>
+        <% if !policy.scap_content.nil? %>
+          <%= display_link_if_authorized policy.scap_content.title,
+                                         hash_for_edit_scaptimony_scap_content_path(:id => policy.scap_content) %>
+        <% end %>
       </td>
       <td>
-          <%= policy.scap_content_profile.nil? ? "Default" : policy.scap_content_profile.title %>
+        <%= policy.scap_content_profile.nil? ? "Default" : policy.scap_content_profile.title %>
       </td>
       <td>
-          <%= action_buttons(
-            link_to_if_authorized(_("Show Guide"), hash_for_scaptimony_policy_path(:id => policy.id)),
-            link_to_if_authorized(_("Edit"), hash_for_edit_scaptimony_policy_path(:id => policy.id)),
-            display_delete_if_authorized(hash_for_scaptimony_policy_path(:id => policy.id),
-              :confirm => _("Delete compliance policy %s with all the reports?") % policy.name)
+        <%= action_buttons(
+                display_link_if_authorized(_("Show Guide"), hash_for_scaptimony_policy_path(:id => policy.id)),
```

Comment:
here the `display_link_if_authroized` is correctly used, because we don't want to display these actions in a button if user is not allowed to do that